ui/src/settings/details/SettingsDetailsView.js

```diff
@@ -56,31 +58,31 @@ const SettingsDetailsView = ({ projectId, ...props }) => {
   }, [fetchXPSettings, props.location.state]);
 
   return (
-    <EuiPage
-      paddingSize="none"
-      className={classNames({ pageWithRightSidebar: isFlyoutVisible })}>
-      <EuiPageBody paddingSize="m">
-        {!isLoaded ? (
-          <EuiTextAlign textAlign="center">
-            <EuiLoadingChart size="xl" mono />
-          </EuiTextAlign>
-        ) : error ? (
-          <EuiCallOut
-            title="Sorry, there was an error"
-            color="danger"
-            iconType="alert">
-            <p>{error.message}</p>
-          </EuiCallOut>
-        ) : (
-          <>
-            {!props["*"].includes("edit") &&
-            !props["*"].includes("segmenters/") ? (
-              <>
-                <EuiPageHeader>
-                  <EuiPageHeaderSection>
-                    <PageTitle icon="managementApp" title="Settings" />
-                  </EuiPageHeaderSection>
-                </EuiPageHeader>
+    <EuiPageTemplate
+      restrictWidth={restrictWidth}
+      paddingSize={paddingSize}
+    >
+      <EuiSpacer size="l" />
+      {!isLoaded ? (
+        <EuiTextAlign textAlign="center">
+          <EuiLoadingChart size="xl" mono />
+        </EuiTextAlign>
+      ) : error ? (
+        <EuiCallOut
+          title="Sorry, there was an error"
+          color="danger"
+          iconType="alert">
+          <p>{error.message}</p>
+        </EuiCallOut>
+      ) : (
+        <>
+          {!props["*"].includes("edit") &&
```

Comment:
Similar comment to above: Could we do `{!props["*"].includes("edit") && !props["*"].includes("segmenters/") && (...)` here instead?